### ISSUE TITLE
fix format varible

### ISSUE THREAD

**Issue by cg-cnu** [abstractfactory/maya-capture#83]
A variable `format` is used which is a reserver keyword in python. 
Change that to something like `output_format`.
Fixed the issue but need to test out, will do a pr soon! 😄

**Comment by mottosso** [comment]
I wouldn't be too fussed about this particular use, partly because it's originally one from `cmds.playblast(format="")` but also because the function referenced by the keyword isn't used in the scope within which it is being used.

In general, if shadowing a keyword improves overall readability, I think it's ok. Put another way, if adhering to PEP08 *reduces* readability, then I'd say it's safe to ignore the particular warning.

**Comment by cg-cnu** [comment]
I agree with you, that's why I left it in the initial pr. Since we are planning for pylinting on travis, thought this might error out. I am not sure if we can tune the pylinter to ignore this case 🙂
